dist/pom.xml

```diff
@@ -432,18 +432,21 @@
                         </goals>
                         <configuration>
                             <artifactItems>
+                                <!-- if add new artifacts, should set `overWrite` as true -->
                                 <artifactItem>
                                     <groupId>com.nvidia</groupId>
                                     <artifactId>spark-rapids-jni</artifactId>
                                     <classifier>${cuda.version}</classifier>
                                     <excludes>META-INF</excludes>
                                     <outputDirectory>${project.build.directory}/parallel-world</outputDirectory>
+                                    <overWrite>true</overWrite>
```

Comment:
nit: the comment above might be more clear if we used a single property with the defualt true in both items:  e.g. dist.spark-rapids-jni.overwrite that could be defined in this pom